### PR TITLE
Fix notebook links

### PR DESCRIPTION
**Pull request by edenlightning**
Todo: add TPU example to the tutorials repo @kaushikb11

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -338,8 +338,8 @@ class LitAutoEncoder(pl.LightningModule):
 ## Examples
 
 ###### Hello world
-- [MNIST hello world](https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/01-mnist-hello-world.ipynb)
-- [MNIST on TPUs](https://colab.research.google.com/github/PytorchLightning/pytorch-lightning/blob/master/notebooks/06-mnist-tpu-training.ipynb)
+- [MNIST hello world](https://pytorch-lightning.readthedocs.io/en/latest/notebooks/lightning_examples/mnist-hello-world.html)
+- [MNIST on TPUs](https://colab.research.google.com/drive/1-_LKx4HwAxl5M6xPJmqAAu444LTDQoa3#scrollTo=BHBz1_AnamN_)
```
Comment (Borda) [inline README.md:342]: Who is the author of this one? the content looks hacked, some cells are invalid

**Review by Borda — CHANGES_REQUESTED** [review]

**Review by justusschock — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

File: README.md (hunk 1 above)
Comment (edenlightning) [reply README.md]: removed this, lets add it when we have it in the tutorials repo

**Review by Borda — APPROVED** [review]

File: docs/source/index.rst [hunk 2]
```diff
@@ -61,6 +61,7 @@ PyTorch Lightning Documentation
    notebooks/lightning_examples/datamodules.ipynb
    notebooks/lightning_examples/cifar10-baseline.ipynb
    notebooks/lightning_examples/basic-gan.ipynb
+   notebooks/lightning_examples/text-transformers.ipynb
    notebooks/lightning_examples/augmentation_kornia.ipynb
```
Comment (Borda) [inline docs/source/index.rst:65]:
```suggestion
   notebooks/lightning_examples/reinforce-learning-DQN.ipynb
   notebooks/lightning_examples/augmentation_kornia.ipynb
```

**Pull request merged**